README.md

```diff
@@ -39,21 +39,12 @@ Read more about Alchemy on the [website](https://alchemy-cms.com) and in the [gu
 - Flexible caching
 - Hostable on any Server that supports Ruby on Rails, a SQL Database and ImageMagick
 
-## 🎮 Demo
-
-Deploy your own free demo on Heroku
-
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/AlchemyCMS/alchemy-demo)
-
-or visit the existing demo at https://alchemy-demo.herokuapp.com
```

Comment:
The demo can be found at 

https://demo.alchemy-cms.com

unfortunately the DNS is not working correctly. CloudFlare has issues with Fly.io shared IPs

For the time being the demo can be accessed at

https://alchemy-demo.fly.dev/

Reply:
Updated the last commit.